src/main/java/Astra.java

```diff
@@ -0,0 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsNotDone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // "X" for done, " " for not done
+    }
+
+    @Override
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + description;
+    }
+}
+
+public class Astra {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Astra!");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        while (true) {
+            String userInput = scanner.nextLine().trim(); // Read and trim input
+
+            if (userInput.equals("bye")) {
+                System.out.println("____________________________________________________________");
+                System.out.println(" Bye. Hope to see you again soon!");
+                System.out.println("____________________________________________________________");
+                break;
+            } else if (userInput.equals("list")) {
+                System.out.println("____________________________________________________________");
+                if (tasks.isEmpty()) {
+                    System.out.println(" No tasks added yet.");
+                } else {
+                    System.out.println(" Here are the tasks in your list:");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println(" " + (i + 1) + ". " + tasks.get(i));
+                    }
+                }
+                System.out.println("____________________________________________________________");
+            } else if (userInput.startsWith("mark ")) {
+                try {
```

Comment:
I like the use of try-catch statements to detect invalid user inputs!
```suggestion
                try {
```